read_command.go

```diff
@@ -88,10 +88,6 @@ func (cmd *readCommand) parseResult(ifc command, conn *Connection) error {
 	}
 
 	if resultCode != 0 {
-		if resultCode == KEY_NOT_FOUND_ERROR {
```

Comment:
Is there another way we can we figure out a key not found error if we return nil here?